views/scss/qti/_hottext.scss

```diff
@@ -28,25 +36,73 @@
             @extend %selectableHover;
         }
 
-        &.error{
-            color : $error;
+        &.error {
+            color: $error;
         }
 
         .hottext-checkmark {
             padding-left: 3px;
             padding-right: 0;
             width: 20px;
+            @include rtl {
+                padding-left: 0px;
+                padding-right: 3px;
+            }
+        }
+
+        .hottext-content {
+            @include rtl {
+                padding-left: 3px;
+            }
         }
 
-        [class^="icon-"], [class*=" icon-"] {
+        [class^="icon-"],
+        [class*=" icon-"] {
             &.small {
                 @include font-size(12);
                 text-shadow: none;
-                line-height:10px;
+                line-height: 10px;
                 top: 1px;
+                &:before {
+                    right: 0em;
+                    @include rtl {
+                        right: -1em;
+                    }
+                }
+                &:after {
+                    left: -1em;
+                    @include rtl {
+                        right: 0em;
+                    }
+                }
             }
-            &.error{
-                color : $error;
+            &.error {
+                color: $error;
+            }
+        }
+    }
+}
+
+// IE10+ CSS styles go here
+@media all and (-ms-high-contrast: none),
```

Comment:
for better readability/understandability/maintainability can we move this hack to mixin?
_qti-base next to @mixin rtl

Reply:
done!